### ISSUE TITLE
Set up platform-console-ui locally

### ISSUE THREAD

**Issue by ndsprinkle** [department-of-veterans-affairs/va.gov-team#45416]
This issue is for setting up the `platform-console-ui` locally to test various backstage plugins. See instructions here: https://github.com/department-of-veterans-affairs/platform-console-ui/blob/main/docs/getting-started/setting-up-the-platform-console-ui.md

**Comment by ndsprinkle** [comment]
All together, I spent about 2.5 days getting everything working locally for the CUI. 

While the set up process on its own is relatively lengthy and complicated, my main slow down was being unable to connect the front end with the back end. This is an issue when attempting to authenticate locally. Without being able to authenticate locally, you pretty much can't use the CUI in any way. 

This issue was ultimately "solved" by doing a mass find and replace in the code for all reference to `127.0.0.1` and changing them to `localhost`. After that, all the docker containers were destroyed and recreated, and then everything finally worked. 

I did not solve this on my own. I sat in a one on one call with someone from the CUI team for about an hour. Despite efforts to log literally every message from the front end and back end, there were no error/warning/info messages of any kind that helped diagnose the problem. The solution that the person helping me ultimately came up with was actually reverting code. Apparently there were some recent changes that switched everything from `localhost` to `127.0.0.1` and switching this back fixed everything. 

Beyond that, the other slow down was requesting the special key/secret values for the `.env` file. This wasn't really that slow, but it is a step that requires a response from someone else, so it's naturally going to fluctuate how much time this adds to the process. I made the mistake of requesting these values towards the end of the day, so I didn't get a response until halfway through the next day. 

The only other thing I can think of that was somewhat slow was setting up Docker. It was mainly slow for me because I had no pre-existing Docker set up and I had to connect Docker to WSL. If a person already has Docker set up locally, then this step would probably be pretty fast.

Speaking of using WSL (Windows), I don't think it slowed anything down. I guess it's possible that my issue with `localhost` and `127.0.0.1` could be related to WSL, but a member of the CUI team uses WSL and has no issues with these changes, so who knows.
